api/src/main/java/info/rmapproject/api/auth/ApiUserServiceImpl.java

```diff
@@ -19,39 +19,44 @@
  *******************************************************************************/
 package info.rmapproject.api.auth;
 
-import info.rmapproject.api.exception.ErrorCode;
```

Comment:
No need to change this for this PR, but something to keep in mind: using IDE auto-formatting, including import optimization, can make it difficult to read diffs.

It's hard to tell if the imports are just being re-ordered because your IDE has different ordering rules, or if the imports have actually changed (e.g. been added to, or removed from).  

So for code that you're editing, it's best to manually optimize the imports (i.e. just delete the lines without re-ordering _all_ the imports).  Often an IDE will have rules, or a configuration preference, regarding re-ordering.  For example, it can be disabled if you want.